src/components/dashboard/ReceiveByQR.web.js

```diff
@@ -129,6 +134,11 @@ const ReceiveByQR = ({ screenProps }) => {
     [showErrorDialog],
   )
 
+  useBrowserSupport({
```

Comment:
only isIOS+non safari should be shown the pop in camera related screensonly isIOS+non safari should be shown the pop in camera related screens